static/css/restyle/restyle.less

```diff
@@ -1516,7 +1516,7 @@ h3.author .transfer-ownership {
     position: relative;
 
     &:nth-of-type(3n) .persona.hovercard:hover {
-      margin-bottom: -3px;
+      margin-bottom: -14px;
```

Comment:
Those 14px look very arbitrary to me in a way that they might look different in different resolutions and stuff but they should work if there's no other way.